webview-ui/src/InspektorGadget/helpers/gadgets.ts

```diff
@@ -100,18 +122,26 @@ function negateSortFunction(fn: SortFunction): SortFunction {
     return (a, b) => -fn(a, b);
 }
 
-function asSortFunction(specifier: SortSpecifier<any>): SortFunction {
-    let key = specifier.property.key;
-    let descending = specifier.direction === SortDirection.Descending;
+function asSortFunction(specifier: SortSpecifier<string>): SortFunction {
+    const key = specifier.property.key;
+    const descending = specifier.direction === SortDirection.Descending;
     const fn: SortFunction = (a, b) => {
-        if (a[key] > b[key]) {
+        const aValue = a[key];
+        const bValue = b[key];
+        if (aValue === null && bValue === null) {
```

Comment:
I'm not sure whether values can ever be `null`. The previous code didn't handle `null`s, so added these extra checks to be safe.